app/src/main/java/de/danoeh/antennapod/activity/OnlineFeedViewActivity.java

```diff
@@ -101,26 +101,24 @@
     private boolean didPressSubscribe = false;
 
     private Dialog dialog;
-    private ListView listView;
-    private Button subscribeButton;
-    private ProgressBar progressBar;
-    private Button stopPreviewButton;
 
     private Disposable download;
     private Disposable parser;
     private Disposable updater;
 
+    private OnlinefeedviewActivityBinding binding;
```

Comment:
Could you please rename to viewBinding? I think that is easier to understand for other contributors